Modules/SubModule_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -23,7 +23,7 @@
 % - 3. Iterate over runs
 % 
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% EXAMPLE:     x = xASL_imp_NII2BIDS_Subject(x, imPar, bidsPar, studyPar, nameSubject);
+% EXAMPLE:     x = xASL_wrp_NII2BIDS_Subject(x, imPar, bidsPar, studyPar, nameSubject);
 % __________________________________
```

Comment:
Can you check the subfunctions and resolve issue #755 as well together with this issue.

Reply:
Don't know if it really makes sense to tackle #755 here. Why not move further with merging this first and doing everything step to step? Was always the right way in the past.

Reply:
OK. No worries, lets do later.